internal/model/beer_media.go

```
@@ -19,81 +20,92 @@ func (bm BeerMedia) TableName() string {
-	if group, exists := countryGroupings[country.Cca3]; exists {
-		return group
+	return Slot{
+		GeoPrefix: s.GeoPrefix,
+		SheetID:   sheetID,
+		SheetSlot: sheetSlot,
 	}
-	if group, exists := regionGroupings[country.Region]; exists {
-		return group
+}
+
+func (s Slot) incrementSheetSlot(sheetSlot string) string {
```

golangci-lint (2.5.0)

[error] unexported method "incrementSheetSlot" for struct "Slot" should be placed after the exported method "String"

(funcorder)